frontend/src/__tests__/cypress/cypress/pages/notebookServer.ts

```diff
@@ -44,6 +44,30 @@ class NotebookServer {
   findAcceleratorProfileSelect() {
     return cy.findByTestId('accelerator-profile-select');
   }
+
+  findOpenInNewTabButton() {
+    return cy.findByRole('button', { name: 'Open in new tab' });
+  }
+
+  findSuccessAlert() {
+    return cy.findByText('Success', { timeout: 120000 });
+  }
+
+  findMinimalPythonImage() {
+    return cy.get(
+      '[data-testid="radio jupyter-minimal-notebook"], [data-testid="radio s2i-minimal-notebook"]',
+    );
+  }
+
+  findPythonVersionsButton() {
+    return cy.get('button[aria-controls*="expandable-section-content"]').contains('Versions');
```

Comment:
@ConorOM1 Maybe there's an option to add `data-testid` to this button - on the frontend side - so Cypress can interact with id instead of `aria-controls*` ?

Reply:
added